providers/base/units/audio/jobs.pxu

```diff
@@ -374,7 +374,7 @@ estimated_duration: 1.0
 imports: from com.canonical.plainbox import manifest
 requires:
   package.name == 'pulseaudio-utils'
-  manifest.has_audio_playback == 'True'
+  audio_card.playback == 'supported'
```

Comment:
It's OK to add this, but I would keep the manifest requirement as well (the same remark applies to the other places where you removed the requirement on `manifest.xxx` below).

Just for your information, if you add the `fail-on-resource` flag to a [job unit](https://checkbox.readthedocs.io/en/latest/reference/units/job.html), the job will fail instead of being skipped if one of the requirements is not met. But in our case, it's a bit annoying, because I guess we would like to raise an alarm if the tester had said "yes, this device has audio playback" (thanks to the `manifest.has_audio_playback == 'True'` bit) and yet the `audio_card.playback` value came back as `unsupported`... but we don't have a way to do that, cause `fail-on-resource` will mark the job as failed if **any** resource requirement is not met.